### ISSUE TITLE
Load balancing

### ISSUE THREAD

**Issue by cavaliercoder** [stonehippo/bullshit-framework#5]
First of all, great project! I use bullshit in almost everything I do now. I'm seeing a lot of other applications on the Internet based on bullshit too so that's great.

One of my clients is currently complaining that the app we're building is too constrained by bullshit. I think it's just a perception problem, but if horizontal scaling was made possible, we could spread bullshit throughout the data center.

Thanks for the good work. I may raise another issue about penetration testing.

**Comment by RobThree** [comment]
Agree, BSF is great but it's not web-scale. We should probably work on that. Maybe  write an optimised filesystem for it.
